config/routes.rb

```diff
@@ -22,6 +22,8 @@
   # route for chatfuel questions and answers
   get 'chatfuel/text_components/:text_component_id/answer_to_question/:index', to: 'chatfuel#answer_to_question', as: 'answer_to_question'
 
+  get '/smaxtec_api/temperature', to: 'smaxtec_api#get_temperature'
```

Comment:
don't add a route, calling the smaxtecs api should not be triggered with a request to the story.board but rather by a cronjob